### PR TITLE
F variables

### PR DESCRIPTION
**Pull request by yvz5**

### CODE REVIEW

File: src/Simplic.Flow.Editor.UI/FlowEditorControl.xaml.cs [hunk 1]
```diff
@@ -63,44 +63,69 @@ public static bool HasImplicitConversion(Type baseType, Type targetType)
...
-                }
-            }
-            else
-            {
-                sourceConnector = null;
-                e.Handled = true;
+                }    
+                
+
+                //if (e.Connector is FlowConnector)
```
Comment (simplicbe) [inline src/Simplic.Flow.Editor.UI/FlowEditorControl.xaml.cs]: Can this be removed?

File: src/Simplic.Flow.Editor.UI/FlowEditorControl.xaml.cs [hunk 2]
```diff
@@ -112,76 +137,78 @@ private void MyDiagram_ConnectionManipulationStarted(object sender, Manipulation
...
-                            diagramViewModel.SourceConnector = null;
-                            diagramViewModel.TargetConnector = null;
-                            return;
-                        }
+            sourceConnector = null;
+            diagramViewModel.SourceConnector = null;
+            diagramViewModel.TargetConnector = null;
+            e.Handled = true;                        
+
+            //if (sourceConnector == null
```
Comment (simplicbe) [inline src/Simplic.Flow.Editor.UI/FlowEditorControl.xaml.cs]: Same here?

File: src/Simplic.Flow.Node/ActionNode/Process/ProcessStartNode.cs [hunk 3]
```diff
@@ -16,14 +16,18 @@ public override bool Execute(IFlowRuntimeService runtime, DataPinScope scope)
                 startInfo.CreateNoWindow = true;
                 startInfo.FileName = scope.GetValue<string>(InPinPath);
```
Comment (yvz5) [inline src/Simplic.Flow.Node/ActionNode/Process/ProcessStartNode.cs]: you should put a file exists check as well

File: src/Simplic.Flow.Node/Simplic.Flow.Node.csproj [hunk 4]
```diff
@@ -1,78 +1,83 @@
...
+    <Compile Include="ActionNode\IO\GetDirectoryContentNode.cs" />
+    <Compile Include="ActionNode\Math\SubtractNode.cs" />
+    <Compile Include="ActionNode\Process\ProcessStartNode.cs" />
+    <Compile Include="ActionNode\Thread\ThreadSleepNode.cs" />
+    <Compile Include="EventNode\IO\OnCheckDirectoryContentEventArgs.cs" />
+    <Compile Include="EventNode\IO\OnCheckDirectoryContentNode.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="ActionNode\Base\SequenceNode.cs" />
+    <Compile Include="ActionNode\Base\StartWithConditionNode.cs" />
+    <Compile Include="YvzNode.cs" />
```
Comment (simplicbe) [inline src/Simplic.Flow.Node/Simplic.Flow.Node.csproj]: Shouldn't we remove this?

File: src/Simplic.Flow.Node/ActionNode/Process/ProcessStartNode.cs (hunk 3 above)
Comment (simplicbe) [reply src/Simplic.Flow.Node/ActionNode/Process/ProcessStartNode.cs]: @yvz5 it is not always a file. Becuse `net start` also works. Or an url...

File: src/Simplic.Flow.Node/Simplic.Flow.Node.csproj (hunk 4 above)
Comment (yvz5) [reply src/Simplic.Flow.Node/Simplic.Flow.Node.csproj]: yes its somehow included again. removing now

**Pull request merged**